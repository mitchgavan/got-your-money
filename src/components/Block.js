import styled from 'styled-components'
import { space, width, fontSize, color, textAlign } from 'styled-system'

export default styled.div`
  ${space}
  ${width}
  ${fontSize}
  ${color}
  ${textAlign}
`
