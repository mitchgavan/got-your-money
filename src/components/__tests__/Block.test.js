import React from 'react'
import Block from '../Block'
import { shallow } from 'enzyme'
import 'jest-styled-components'
import toJson from 'enzyme-to-json'

it('renders correctly', () => {
  const wrapper = shallow(<Block color="red" bg="white" />)
  expect(toJson(wrapper)).toMatchSnapshot()
})
