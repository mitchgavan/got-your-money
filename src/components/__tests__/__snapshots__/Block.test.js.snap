// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.c0 {
  padding: 12px;
  width: 200px;
  font-size: 16px;
  color: red;
  background-color: #fefefe;
  text-align: center;
}

<div
  className="c0"
  color="red"
  fontSize={2}
  width={200}
/>
`;
